bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/discovery/EnOceanDeviceDiscoveryService.java

```diff
@@ -183,8 +179,9 @@ public void eventReceived(EventMessage event) {
         }
     }
 
-    private Integer sendTeachInResponse(ERP1Message msg, String enoceanId) {
+    private @Nullable Integer sendTeachInResponse(ERP1Message msg, String enoceanId) {
         // get new sender Id
+        @Nullable
```

Comment:
The compiler is intelligent enough on local variables, so this could be removed.